neptune/alpha/internal/init_impl.py

```diff
@@ -50,39 +54,50 @@ def init(
 
     if not project:
         project = os.getenv(PROJECT_ENV_NAME)
+    if connection_mode == 'offline':
+        project = 'offline-project-placeholder'
     if not project:
         raise MissingProject()
 
-    credentials = Credentials()
-
     if connection_mode == "async":
         # TODO Initialize backend in async thread
-        backend = HostedNeptuneBackend(credentials)
+        backend = HostedNeptuneBackend(Credentials())
```

Comment:
This is really strange - tests fail locally and on Travis with

```
        if connection_mode == "async":
            # TODO Initialize backend in async thread
            backend = HostedNeptuneBackend(Credentials())
        elif connection_mode == "sync":
>           backend = HostedNeptuneBackend(Credentials())
E           TypeError: __init__() takes 1 positional argument but 2 were given
```

and they pass when I change this line to `backend = HostedNeptuneBackend()`.

But clearly `HostedNeptuneBackend` takes one (that is, two, including `self`) parameters:

```
class HostedNeptuneBackend(NeptuneBackend):

    def __init__(self, credentials: Credentials, proxies: Optional[Dict[str, str]] = None):
        self.credentials = credentials
```

@aniezurawski do you have an idea what is happening here?